layers/gd/Dockerfile

```diff
@@ -1,50 +1,16 @@
+# syntax = docker/dockerfile:1.4
 ARG PHP_VERSION
-FROM bref/build-php-$PHP_VERSION:1.7.4 AS ext
+FROM bref/build-php-$PHP_VERSION:2 AS ext
 
-# Install required build libraries
-RUN set -ue \
-    ; LD_LIBRARY_PATH= \
-    ; yum -y install \
-    libwebp-devel \
-    libXpm-devel \
-    libpng-devel \
-    libjpeg-devel \
-    freetype-devel \
-    ;
-
-WORKDIR ${PHP_BUILD_DIR}/ext/gd
-RUN phpize
-ARG PHP_VERSION
-ENV EXT_CONFIGURE_OPTS=" \
-    --with-freetype \
-    --enable-gd \
-    --with-jpeg \
-    --with-png \
-    --with-webp \
-    --with-xpm \
-    --with-zlib \
-    "
-RUN ./configure \
-    --disable-static \
-    --enable-gd-jis-conv \
-    --enable-shared \
-    ${EXT_CONFIGURE_OPTS} \
-    ;
-RUN make -j $(nproc)
-RUN make install
-RUN cp "$(php-config --extension-dir)/gd.so" /tmp/gd.so
-RUN echo 'extension=/opt/bref-extra/gd.so' > /tmp/ext.ini
+RUN <<eot bash -e
+yum install -y php-gd
+echo 'extension=gd.so' > /tmp/ext-gd.ini
+eot
 
+# Build the final image with just the files we need
 FROM scratch
 
-COPY --from=ext /usr/lib64/libXpm.so.4 /opt/bref/lib64/libXpm.so.4
```

Comment:
I think you'll need to keep most of these for gd to work properly.

Messing around on a local branch I had to add the following for it to load
```
COPY --link --from=ext /usr/lib64/libgd.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libXpm.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libX11.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libpng15.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libjpeg.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libfreetype.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libfribidi.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libglib-2.0.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libgraphite2.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libharfbuzz.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libjbig.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libraqm.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libtiff.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libxcb.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libXau.so* /opt/lib/
COPY --link --from=ext /usr/lib64/libwebp.so* /opt/lib/
COPY --link --from=ext /usr/lib64/php/modules/gd.so* /opt/bref/extensions/
COPY --link --from=ext /tmp/*.ini /opt/bref/etc/php/conf.d/
```